src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -104,11 +104,18 @@ def add_helm_repo(kube_config, kube_context):
         raise CLIError("Unable to add repository {} to helm: ".format(repo_url) + error_helm_repo.decode("ascii"))
 
 
-def get_helm_registry(cmd, location):
+def get_helm_registry(cmd, location, dp_endpoint_dogfood=None, release_train_dogfood=None):
+    # Setting uri
     get_chart_location_url = "https://{}.dp.kubernetesconfiguration.azure.com/{}/GetLatestHelmPackagePath?api-version=2019-11-01-preview".format(location, 'azure-arc-k8sagents')
     release_train = os.getenv('RELEASETRAIN') if os.getenv('RELEASETRAIN') else 'stable'
+    if dp_endpoint_dogfood:
+        get_chart_location_url = "{}/azure-arc-k8sagents/GetLatestHelmPackagePath?api-version=2019-11-01-preview".format(dp_endpoint_dogfood)
+        if release_train_dogfood:
+            release_train = release_train_dogfood
     uri_parameters = ["releaseTrain={}".format(release_train)]
-    resource = cmd.cli_ctx.cloud.endpoints.management
+    resource = cmd.cli_ctx.cloud.endpoints.active_directory_resource_id
```

Comment:
>active_directory_resource_id [](start = 43, length = 28)

what is this change?

Reply:
This is the resource endpoint which works for both the clouds. The value of management endpoint fails to contact dp in case of Dogfood